js/gateio.js

```diff
@@ -1432,7 +1432,8 @@ module.exports = class gateio extends Exchange {
         const request = {};
         const futures = type === 'futures';
         const swap = type === 'swap';
-        if ((swap || futures) && !params['settle']) {
+        const settle = this.safeString (params, 'settle')
```

Comment:
```suggestion
        const settle = this.safeString (params, 'settle');
```

Linting error: missing semicolon

Reply:
Good catch, thanks!